### PR TITLE
Adds new Statsig events to login page

### PR DESCRIPTION
**Pull request by hannahbergam**
Adds two new events to the Login page as seen below. I was able to test both locally and with localmode turned off. I am calling these 'login' instead of 'signin' because we as a team seem to frequently misuse/mixup 'signin' and 'signup'. 

Screenshot 2024-09-12 at 1 55 36 PM
Screenshot 2024-09-12 at 3 24 32 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-2387

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/webpackEntryPoints.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ const CODE_STUDIO_ENTRIES = {
   'programming_expressions/show': './src/sites/studio/pages/programming_expressions/show.js',
   'sessions/lockout': './src/sites/studio/pages/sessions/lockout.js',
   'devise/sessions/new': './src/sites/studio/pages/devise/sessions/new.js',
+  'devise/sessions/_login': './src/sites/studio/pages/devise/sessions/_login.js',
```
Comment (bethanyaconnor) [inline apps/webpackEntryPoints.js:57]: wow I'm surprised we didn't have JS on that page 😅

File: dashboard/app/views/devise/sessions/_login.html.haml [hunk 2]
```diff
@@ -1,3 +1,7 @@
+
+- content_for :head do
```
Comment (bethanyaconnor) [inline dashboard/app/views/devise/sessions/_login.html.haml:2]: I know you've explained this multiple times but remind me why this needs to be in the head? (I'm getting all turned around lol)

**Review by bethanyaconnor — APPROVED** [review]

File: apps/webpackEntryPoints.js (hunk 1 above)
Comment (hannahbergam) [reply apps/webpackEntryPoints.js:57]: Right!?

File: dashboard/app/views/devise/sessions/_login.html.haml (hunk 2 above)
Comment (hannahbergam) [reply dashboard/app/views/devise/sessions/_login.html.haml:2]: I've started adding any Statsig JS in the head of the document so it loads first especially when the page might be the first a user visits (and we would want to make sure Statsig has time to initialize). I was imagining this is the kind of page could be linked in SEO, so am adding to the head!

**Pull request merged**